File: Sistema-de-Gerenciamento-de-Eventos-e-Inscricoes/API/Models/Evento.cs
using System;

namespace API.Models;

public class Evento
{
    public int Id { get; set; }
    public string? Nome { get; set; }
    public string? Descricao { get; set; }
    public DateTime DataEvento { get; set; }
    public int VagasMaximo { get; set; }
    public int VagasRestantes { get; set; }
    public Usuario? Proprietario { get; set; }
    public int ProprietarioId { get; set; }
}

File: front/src/models/Evento.ts
import { Usuario } from "./Usuario";

export interface Evento {
    id? : number;
    nome : string;
    descricao : string;
    dataEvento : string;

    proprietarioId : number;
    proprietario? : Usuario;

    vagasRestantes? : number;
    vagasMaximo : number;
}